src/main/resources/db/migration/common/V1_280__add_regions.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO region (reg_id, region_name) VALUES(1, 'London');
+INSERT INTO region (reg_id, region_name) VALUES(2, 'South East');
+INSERT INTO region (reg_id, region_name) VALUES(3, 'South West');
+INSERT INTO region (reg_id, region_name) VALUES(4, 'Midlands');
+INSERT INTO region (reg_id, region_name) VALUES(5, 'North East');
+INSERT INTO region (reg_id, region_name) VALUES(6, 'North West');
+INSERT INTO region (reg_id, region_name) VALUES(7, 'Wales');
+
+ALTER SEQUENCE aut_seq RESTART WITH 8;
```

Comment:
This should be `reg_seq` - although it's odd that we have a sequence for a standing data table.
But hey ho - may as well bump it whilst it's here

Reply:
Hia Richard. Believe their are issues with the sequences not being set between integration tests when the databases are cleared.